lib/cosi/cosi.go

```diff
@@ -191,15 +191,25 @@ func (c *Cosi) CreateResponse() (*Response, error) {
 
 // Response generates the response from the commitment, challenge and the
 // responses of its children.
-func (c *Cosi) Response(responses []*Response) (*Response, error) {
-	// create your own response
-	if err := c.genResponse(); err != nil {
-		return nil, err
+// if respond == true, will also add our response
+func (c *Cosi) Response(respond bool, responses []*Response) (*Response, error) {
```

Comment:
I would prefer putting the bool flag at the end of the list arguments: `Response(responses []*Response, respond bool)` as to keep consistent with the `Commit(commits []*Commitment)` function.